src/add-search-spec.js

```diff
@@ -53,13 +54,20 @@ describe('Search Results', () => {
   });
 
   it('should generate search results', (done) => {
-    let config = JSON.parse(browser.params.skyPagesConfig);
+    let config = browser.params.skyPagesConfig;
+    let doesSearchConfigExist = (
+      config.skyux &&
+      config.skyux.appSettings &&
+      config.skyux.appSettings.stache &&
+      config.skyux.appSettings.stache.searchConfig
+    );
     let siteName = config.skyux.name;
     if (!siteName) {
       const packageFile = require('../package.json');
       siteName = packageFile.name;
     }
     let url = config.skyux.host.url;
+    let isInternal = doesSearchConfigExist ? config.skyux.appSettings.stache.searchConfig.is_internal : true;
```

Comment:
I'm curious why the property uses an underscore instead of camel case?

Reply:
So for the properties that are only used by the skyuxconfig, I use camel case. For .net services, the standard is snake case for some reason, so I am using snake case for all properties that get pushed to the backend as well. It is a bit weird, but it helps me keep things that are relevant to my scraped objects separate from config that's only for the SPA and cli commands. Perhaps I should just have them all be camel case for consistency, but that was my initial thought behind it.